### DISCUSSION TITLE
Docker Dev version updates - What's changed?

### DISCUSSION THREAD

**Discussion by Sboshoff76** [celsworth/lxp-bridge#224] [General]
I see the dev version was updated a few days ago. How can I see what changes were made there?

PS. I'm not a developer so can't look at the code changes to figure it out.

**Comment by celsworth** [comment]
There's a changelog here - https://github.com/celsworth/lxp-bridge/blob/master/CHANGELOG.md - that should also be linked in the HA addon info.

Anything in "unreleased" is in the dev version; anything in a named version (like 0.13.0) is in the stable release.

**Comment by Sboshoff76** [comment]
Understood.. Thank you!
